client/landing/stepper/declarative-flow/internals/steps-repository/link-in-bio-setup/styles.scss

```diff
@@ -0,0 +1,140 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+$gray-color: var( --studio-gray-20 );
+$light-gray-color: var( --studio-gray-0 );
+$light-blue-color: var( --studio-blue-5 );
+$dark-blue-color: var( --studio-blue-80 );
+$input-error-color: var( --studio-red-60 );
+$input-border-color: var( --studio-gray-10 );
+$get-started-button-primary-color: var( --studio-blue-50 );
+$white: var( --color-surface );
```

Comment:
As you can see [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/preview-toolbar.scss#L37), you can directly use the var 👍